app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "shared/nav", user: @user %>
+<h1 class="recipes_header">Recipes</h1>
+<div class="recipe">
+  <%= link_to "New recipe", new_recipe_path, class: "foods-add-button" %>
+  <br>
+  <br>
+  <div id="recipes">
+    <% @recipes.each do |recipe| %>
+      <div class="each-recipe">
+        <% if can? :read, recipe %>
+          <h3 class="nav-links"><%= link_to "#{recipe.name}", recipe_path(recipe.id), class: "links" %></h3>
+          <p><%= recipe.description %></p>
+          <% if can? :destroy, recipe %>
+            <%= button_to "Delete recipe",
+            recipe_path(recipe.id),
+            class: "recipe-delete-button",
+            method: :delete %>
+          <% end %>
+        <% end %>
+      </div>
```

Comment:
- [ ] kindly match the recipe cards to the given wireframe
in your design :point_down: 
![Screenshot from 2023-06-29 12-05-08](https://github.com/Ade179/wine-and-dine/assets/66097371/f8ca7a5e-fec6-41fd-a1b1-84dfce6b1d66)
wireframe :point_down: 
![Screenshot from 2023-06-29 12-04-57](https://github.com/Ade179/wine-and-dine/assets/66097371/dc666ee9-8997-4277-9457-9639ac6cad4f)